uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenDropInService.kt

```diff
@@ -51,9 +47,34 @@ class AdyenDropInService : DropInService() {
 
     // Handling for submitting additional payment details
     override fun makeDetailsCall(actionComponentData: JSONObject): CallResult {
+        val payload = JSONObject()
```

Comment:
Would be nice to move the logic outside this class to ensure SRP, could be a presenter/view and treat this class as the View. 
I understand we are trying to make it work and it is easier to keep it here for the time being

Reply:
Yes. Agreed. Still not sure how everything fits together so keeping it here for now.